### DISCUSSION TITLE
Modern Search v4 - April 2022 (pre-release)

### DISCUSSION THREAD

**Discussion by FranckyC** [microsoft-search/pnp-modern-search#1982] [Announcements]
# PnP Modern Search v4

**This solution includes now only one `.sppkg` file:**
- `pnp-modern-search-parts-v4.sppkg`. Mandatory.

**Getting started documentation here:** [https://microsoft-search.github.io/pnp-modern-search/](https://microsoft-search.github.io/pnp-modern-search/)

[Feedback and comments](https://github.com/microsoft-search/pnp-modern-search/issues) are welcomed!

### New features

- PR #1931
     - Added a sort control component to layouts:
          - Now the list of sortable properties is a static list of 'Sortable' managed properties in the search schema. Custom properties or RefinableXXX have to be entered manually.
          - Updated the Details List sort behavior. Now there is no static sorting anymore. The field used to sort a column comes from the data source sort settings configuration and we clearly make the distinction between display value and sort value in the configuration.
- PR #1873 - Included new function for Handlebars Helper.

### Fixes

- PR #1980 - Fixed #1840 Added a conditional check for the data source reference property.
- PR #1979 - Fixed #1965 Added properties and utils from the main context to the Details List and Cards Handlebars context.
- PR #1952 - Fixed #1893 Added note in documentation about SPFx version for extensions.
- PR #1951 - Fixed #1943 Combo box search filter is now case insensitive.
- PR #1937 - Added data-instance-id to pnp-slider template.
- PR #1879 - Fixed #1878 Fix for multi-value UPA values.
- PR #1837 - Fixed #1797 Only enclosed term text value with the string() operator. Other values remain HEX encoded.
- PR #1836 - Fixed #1835. Refinement token shouldn't be enclosed with quotes if contains the range() FQL operator.

## Search filters

N/A

## Search Verticals

N/A

## Search Box

N/A

## Key contributors

- @wobba 
- @FranckyC 
- @gszdev
- @kachihro
- @edarroudi

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.6.0-beta'>Modern Search v4 - April 2022 (pre-release)</a>.</em>

**Comment by edarroudi** [comment]
Hey @FranckyC , thank you for your hard work.
Would it be possible to add PR #1786  to this release? It's a small, tested PR and I didn't see a reason to not add it - sorry if I'm too demanding.

Also, if it's ok, i would take over the german translation from #1541 - I'm a german as well so that would just take a couple of minutes to add the missing translations.

Thank you again!

**Reply by FranckyC** [reply]
Just missed it sorry. I see there are some conflicts on the doc for #1786, could you resolve them? I'll try to include the PR into the next release. We are only on pre release now so we still have time to adjust.

Also if you can update the translations for #1541, sure go ahead, we will integrate them as well (start from English as reference and don't forget to do all Web Parts + common translations).

**Reply by edarroudi** [reply]
No worries. I fixed #1786  and updated the translations in another PR #1987 , based on #1541. I think I don't have permission to adjust #1541 directly as I'm no maintainer.

**Reply by FranckyC** [reply]
Merged #1786 and #1987
